RelayTestApp/Assets/Scripts/Data/Lobby.cs

```diff
@@ -24,7 +24,7 @@ public Lobby(Dictionary<string, object> lobbyJson, string in_lobbyId)
             var user = new UserInfo(jsonMember);
             if (user.ID == GameManager.Instance.CurrentUserInfo.ID)
             {
-                user.AllowSendTo = false;
+                GameManager.Instance.CurrentUserInfo = user;
```

Comment:
What happens to `user.AllowSendTo = false;` ? I'm not too familiar with this projects code but I don't understand what this change is doing

Reply:
I was making the local user not update their own cursor position but then I made a change for that to include the local user for consistency between the platforms. The "AllowSendTo" was a bool I would use to determine that previously to update the cursor position or not.

Reply:
Ah gotcha, makes sense, thanks!